homeassistant/components/zha/__init__.py

```diff
@@ -8,8 +8,8 @@
 from homeassistant.helpers.device_registry import CONNECTION_ZIGBEE
 
 # Loading the config flow file will register the flow
-from . import config_flow  # noqa: F401 pylint: disable=unused-import
```

Comment:
This import needs to stay here, see https://github.com/home-assistant/home-assistant/pull/29555#issuecomment-562700593 for more details on this

Reply:
No, it should not stay there, this issue needs to be addressed 😃 
@Adminiuga @dmulcahey Any insight on this one?

Reply:
Yeah you are right 😁 @Adminiuga said that he is looking into this, see https://github.com/home-assistant/home-assistant/pull/29555#issuecomment-562738712

Reply:
@frenck IIRC this is to get the config_flow file to be loaded in order to get the flow handler registered to the config entry manager. Not sure if the manifest together with FlowHandler(config_entries.ConfigFlow, domain=DOMAIN) improves that situation. But this has been a requirement since @balloob wrote the config entries

Reply:
Ok, let's add `# isort:skip` to that line and put it back to its original location. @basnijholt, can you do that?

Reply:
Let's add `# isort:skip` for now, but you are right, we're need to address it. It's on my list.

Reply:
Circular dependency at `api.py` -> `core/__init__.py` -> `core/gateway.py` -> `api.py`. The problem is that `core/const.py` causes `core/__init__.py` to be processed. 

However I can't reproduce it with `python3 -c "import homeassistant.components.zha.api`, which is weird.

Reply:
You're right, it is circular dependency in `zha/api.py` is causing this issue. It's a learning curve there for me :) I was thinking moving the `const.py` one dir up would fix it, just need to find time to finish it.

Reply:
btw, we no longer need to import the config flow in `__init__.py` to register it

Reply:
@balloob I have removed the import in dbda0774a9c7308611d9205590b3f491272bfc1f. Hopefully, this fixed the problem.

Reply:
@basnijholt For this PR, we should only move back the highlighted line by this comment, back to its original location with an isort ignore.
We should not do multiple/different changes in this PR, that is out of scope, and it will start becoming really hard to keep track of it all real quick.

So let's keep things simple, and just revert this single line, and add an ignore.

Reply:
Ah, that didn't work. 

Can anyone else fix this? Because I don't know how to.